index.js

```diff
@@ -125,7 +125,16 @@ function createSeries (opts, archs, platforms) {
               fs.mkdirs(buildDir, cb)
             },
             function (cb) {
-              extract(zipPath, {dir: buildDir}, cb)
+              extract(zipPath, {dir: buildDir}, function (err) {
+                if (err || !opts.afterExtract || !Array.isArray(opts.afterExtract)) {
+                  cb(err)
+                } else {
+                  var newFunctions = opts.afterExtract.map(function (fn) {
+                    return fn.bind(this, buildDir, version)
+                  })
+                  series(newFunctions, cb)
+                }
+              })
```

Comment:
I think the callback you defined for `extract` be another function in the series.